### ISSUE TITLE
ICS devices fail randomly with "ABORTING: INVALID HEAP ADDRESS IN dlfree"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#10]
```
What steps will reproduce the problem?
1. ICS Device fail randomly
2.
3.

What is the expected output? What do you see instead?
Limbo crashes at g_free, this is probably due to optimization.

Please use labels and text to provide additional information.
Try optimization -O2 and/or remove any optimization for glib build

```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 6 Oct 2012 at 8:22

**Comment by GoogleCodeExporter** [comment]
```
Error log, disable striping of debug symbols to get more information on g_free()

F/libc (29409): @@@ ABORTING: INVALID HEAP ADDRESS IN dlfree
F/libc (29409): Fatal signal 11 (SIGSEGV) at 0xdeadbaad (code=1)
```

Original comment by `ideal2i...@gmail.com` on 6 Oct 2012 at 8:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `ideal2i...@gmail.com` on 6 Oct 2012 at 8:25
- Changed title: **ICS devices fail randomly with "ABORTING: INVALID HEAP ADDRESS IN dlfree"**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 0.9.2
```

Original comment by `ideal2i...@gmail.com` on 11 Oct 2012 at 4:05
- Changed state: **Fixed**
